app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -198,6 +199,13 @@ module.exports = function (defaults) {
     packagerOptions: {
       webpackConfig: {
         devtool: "source-map",
+        resolve: {
+          alias: {
+            // This is a build-time alias is for code in core only – plugins
+            // and legacy bundles go through the runtime loader.js shim
+            I18n: "discourse-i18n",
+          },
+        },
```

Comment:
I thought `autoImport.alias` above is supposed to work, but it didn't. Seems like that doesn't do anything under embroider.